src/Donations/Controllers/DonationRequestController.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+namespace Give\Donations\Controllers;
+
+use Give\Donations\Models\Donation;
+use Give\Donations\ValueObjects\DonationRoute;
+use phpDocumentor\Reflection\Types\Boolean;
+use WP_Error;
+use WP_REST_Request;
+use WP_REST_Response;
+
+/**
+ * @unreleased
+ */
+class DonationRequestController
+{
+    /**
+     * @var Boolean
+     */
+    private $isAdmin;
+
+    /**
+     * @var Boolean
+     */
+    private $isAnonymousDonationsRedacted;
```

Comment:
I recommend using local variables and parameters rather than class properties, as these are really part of the handling of a single request flow, rather than a distinct attribute of the class itself.

Reply:
Resolved in 06c849945a5d83d494ce824490fb727cf46e427a